src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,110 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private T[] elements;
+    private int size;
+    private int currentCapacity = 10;
+
+    public ArrayList() {
+        elements = (T[]) new Object [currentCapacity];
+    }
+
+    private void increaseCapacity() {
```

Comment:
Be attantive
![image](https://user-images.githubusercontent.com/20326858/218323573-9958b7b4-9251-4d80-b575-43b310cccda7.png)

Reply:
not fixed